ttnn/cpp/ttnn/operations/data_movement/fold/fold.cpp

```diff
@@ -4,25 +4,29 @@
 
```

Comment:
Did you change anything in this file? If not, please undo the changes.

Reply:
Resolved in last commit.